web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -168,15 +210,33 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
       return;
     }
 
-    await removeReviewPeriod(
-      toDelete,
-      csrf
-    );
-    dispatch({ type: UPDATE_REVIEW_PERIODS, payload: periods.filter((period) => period?.id !== toDelete) });
+    await removeReviewPeriod(toDelete, csrf);
+    dispatch({
+      type: UPDATE_REVIEW_PERIODS,
+      payload: periods.filter(period => period?.id !== toDelete)
+    });
     handleConfirmClose();
   }, [csrf, periods, dispatch, toDelete, handleConfirmClose]);
 
+  const loadFeedbackTemplates = useCallback(async () => {
```

Comment:
This function is new. It gets all the available feedback templates so they can be displayed in drop-downs.